src/apps/commands/management/commands/generate_data.py

```diff
@@ -0,0 +1,59 @@
+import random
+
+from django.core.management.base import BaseCommand
+from competitions.models import CompetitionParticipant
+from factories import UserFactory, CompetitionFactory, PhaseFactory, SubmissionFactory
+
+
+class Command(BaseCommand):
+    help = "Generate data for testing"
+
+    def add_arguments(self, parser):
+        # Optional argument
+        parser.add_argument(
+            '-s'
+            '--size',
+            type=int,
+            help='Determines the size of data to be created, i.e. 5 users, each with 5 comps with 5 phases with 5 submissions.'
+                 'This defaults to 5')
+
+    def handle(self, *args, **kwargs):
+        size = kwargs['size'] if 'size' in kwargs else 5
+        users = [UserFactory(username='admin', super_user=True) if i == 0 else UserFactory() for i in range(size)]
+        competition_count = 0
```

Comment:
do we need all of these counters? aren't they deterministic based off of input size?

Reply:
Yea, I can pull them. I had them there to verify the number of things being generated was as I expected. Since it is nested, it is exponential (i.e. size 5 is 5 users but 625 submissions), and things weren't coming out right, made for helpful error checking. But also I liked seeing how many of each thing were actually created, mostly for funziez.

Reply:
- [x] done